x-pack/plugins/cases/public/components/user_actions/comment/registered_attachments.test.tsx

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+
+import type {
+  AttachmentType,
+  CommonAttachmentViewProps,
+} from '../../../client/attachment_framework/types';
+import { AttachmentActionType } from '../../../client/attachment_framework/types';
+import { AttachmentTypeRegistry } from '../../../../common/registry';
+import { getMockBuilderArgs } from '../mock';
+import { createRegisteredAttachmentUserActionBuilder } from './registered_attachments';
+
+const getLazyComponent = () =>
+  React.lazy(() => {
+    return Promise.resolve().then(() => {
+      return {
+        // eslint-disable-next-line react/display-name
+        default: React.memo(() => {
+          return <>{'My component'}</>;
+        }),
+      };
+    });
+  });
+
+describe('createCommentUserActionBuilder', () => {
```

Comment:
nit: should this be `createRegisteredAttachmentUserActionBuilder`?

`createCommentUserActionBuilder` is in `x-pack/plugins/cases/public/components/user_actions/comment/comment.test.tsx`

Reply:
Copy pasta 🙂